synapse/module_api/__init__.py

```diff
@@ -50,11 +51,17 @@ def __init__(self, hs, auth_handler):
         self._auth = hs.get_auth()
         self._auth_handler = auth_handler
         self._server_name = hs.hostname
+        self._presence_stream = hs.get_event_sources().sources["presence"]
 
         # We expose these as properties below in order to attach a helpful docstring.
         self._http_client = hs.get_simple_http_client()  # type: SimpleHttpClient
         self._public_room_list_manager = PublicRoomListManager(hs)
 
+        # The next time these users sync, they will receive the current presence
+        # state of all local users. Users are added by send_local_online_presence_to,
+        # and removed after a successful sync.
+        self.send_full_presence_to_local_users = set()
```

Comment:
```suggestion
        self._send_full_presence_to_local_users = set()
```

I think we should only have private attrs here, to avoid plugins ever thinking its a good idea to mess with the attributes directly?

Reply:
Hmm true, though we do need to access this from other bits of Synapse. That's probably fine if we add a comment.

An alternative is to make this a global variable at the file level?

Reply:
Have made it private.